components/raftstore/src/store/config.rs

```diff
@@ -60,9 +60,11 @@ pub struct Config {
     // When the entry exceed the max size, reject to propose it.
     pub raft_entry_max_size: ReadableSize,
 
-    // Interval to gc unnecessary raft log (ms).
+    // Interval to gc unnecessary raft log (s).
     pub raft_log_gc_tick_interval: ReadableDuration,
     // A threshold to gc stale raft log, must >= 1.
+    // Note: this config is not needed after #8668, but some tests rely on it.
+    #[doc(hidden)]
```

Comment:
Just a note, only need to keep the deprecated item here if we need to examine it during validation, e.g. inherit a renamed item value, yield a warning. The parsing process itself works just fine with unrecognized items.

Reply:
I'm not sure if the upgrade tool will automatically reduce the deprecated config items. If it won't, TiKV will see an unrecognised config, raise an error and exit.

Reply:
Did you verify that? I ran an unit test before and it didn't raise error.

Reply:
I believe tikv currently just ignore unrecognized configs.